data/predefined_catalogs.yml

```diff
@@ -38,9 +38,8 @@ aws_data:
     v2023.2: 897e5c5272875f1c066f393798b7ae59721c9e9d
   notes: This data are stored in public Amazon Web Services.
 artifact_data:
-  urlpath: https://github.com/DirkEilander/hydromt-artifacts/releases/download/{version}/data.tar.gz
+  urlpath:  https://raw.githubusercontent.com/Deltares/hydromt/{version}/data/catalogs/artifact_data.yml
   versions:
-    v0.0.8: v0.0.8
-    v0.0.7: v0.0.7
-    v0.0.6: v0.0.6
+    v0.0.9: main
+    v0.0.8: 202874eb4fe3415d0608ea81cd61620af6f5816a
```

Comment:
This will break older versions of hydromt I think. So unless very necessary I think this is more something for v1